packages/react/src/components/Notification/Notification.js

```diff
@@ -499,6 +506,11 @@ InlineNotification.propTypes = {
    */
   iconDescription: PropTypes.string.isRequired,
 
+  /**
+   * Provide a description for "status" icon that can be read by screen readers
+   */
+  statusIconDescription: PropTypes.string,
```

Comment:
```suggestion
  statusIconDescription: PropTypes.string.isRequired
```

does this also need to be required like `iconDescription`?

Reply:
Yeah makes sense to be consistent 👍